core/src/local_cluster.rs

```diff
@@ -142,12 +142,12 @@ impl LocalCluster {
             cluster.add_validator(&fullnode_config, *stake);
         }
 
+        discover(&cluster.entry_point_info.gossip, node_stakes.len()).unwrap();
```

Comment:
btw this fixes some flakyness i experienced locally with replicator tests.

Reply:
cool